src/stirling/source_connectors/socket_tracer/uprobe_manager.cc

```diff
@@ -654,7 +648,7 @@ int UProbeManager::DeployOpenSSLUProbes(const absl::flat_hash_set<md::UPID>& pid
       // before the BPF map is updated. This value is cleaned up when the upid is
       // terminated, so if attachment fails it will be deleted prior to the pid being
       // reused.
-      openssl_source_map_->UpdateValue(pid.pid(), kStaticallyLinkedSource);
+      PX_UNUSED(openssl_source_map_->SetValue(pid.pid(), kStaticallyLinkedSource));
```

Comment:
This is what I would prefer to do with the other `openssl_source_map_->SetValue` calls. Regardless of what we pick, I think it's best to treat all of them the same.